models/user.go

```diff
@@ -1237,25 +1238,25 @@ func SearchUserByName(opts *SearchUserOptions) (users []*User, _ int64, _ error)
 
 	searchQuery := "%" + opts.Keyword + "%"
 	users = make([]*User, 0, opts.PageSize)
+
 	// Append conditions
-	sess := x.
-		Where("LOWER(lower_name) LIKE ?", searchQuery).
-		Or("LOWER(full_name) LIKE ?", searchQuery).
-		And("type = ?", opts.Type)
+	cond := builder.Eq{
+		"type": opts.Type,
+	}.
+		And(builder.Like{"lower_name", searchQuery}.
+			Or(builder.Like{"LOWER(full_name)", searchQuery}))
```

Comment:
It will be like as the following if more one `And` condition? :(

```
And(builder.Like{"lower_name", searchQuery}.
  And(builder.Like{"lower_name", searchQuery}.
    And(builder.Like{"lower_name", searchQuery}.
      And(builder.Like{"lower_name", searchQuery}.
        Or(builder.Like{"LOWER(full_name)", searchQuery}))
```